packages/grafana-ui/src/utils/handleReduceMotion.ts

```diff
@@ -0,0 +1,11 @@
+import { CSSInterpolation } from '@emotion/css';
+
```

Comment:
think we could also add some JSDoc here about how to use this function and what the args are